# Chemistry I
## Course code: CH 1101
- Credits: 3
- Program: BTech
- Course Status: Core

## Tutorial Sheet 1

## Working with `scipy.constants`

In [2]:
from scipy.constants import c             # speed of light
from scipy.constants import m_e           # mass of electron
from scipy.constants import h             # Planck's constant
from scipy.constants import N_A           # Avogadro constant
from scipy.constants import k             # Boltzmann constant
from scipy.constants import e             # elementary charge
from scipy.constants import Rydberg as Ry # Rydberg constant
from scipy.constants import eV            # one electron volt in Joules

print( 'c = {} m s^-1'.format( c ) )
print( 'm_e = {} kg'.format( m_e ) )
print( 'h = {} J s'.format( h ) )
print ( 'N_A = {} mol^-1'.format (N_A) )
print( 'k = {} J K^-1'.format( k ) )
print( 'e = {} C'.format(e) )
print( 'Ry = {} m^-1'.format(Ry) )
print( 'eV = {} J'.format(eV) )

c = 299792458.0 m s^-1
m_e = 9.1093837015e-31 kg
h = 6.62607015e-34 J s
N_A = 6.02214076e+23 mol^-1
k = 1.380649e-23 J K^-1
e = 1.602176634e-19 C
Ry = 10973731.56816 m^-1
eV = 1.602176634e-19 J


#### Physical constants
`scipy.constants` also contains a `physical_constants` database (as a Python dictionary)
that contains an even larger number of constants, and also gives their units.

In [3]:
from scipy.constants import physical_constants

print( physical_constants['speed of light in vacuum'] )
print( physical_constants['electron mass'] )
print( physical_constants['Planck constant'] )
print( physical_constants['Avogadro constant'] )
print( physical_constants['Boltzmann constant'] )
print( physical_constants['elementary charge'] )
print( physical_constants['proton mass'] )
print( physical_constants['Rydberg constant'] )
print( physical_constants['electron volt'] )

(299792458.0, 'm s^-1', 0.0)
(9.1093837015e-31, 'kg', 2.8e-40)
(6.62607015e-34, 'J Hz^-1', 0.0)
(6.02214076e+23, 'mol^-1', 0.0)
(1.380649e-23, 'J K^-1', 0.0)
(1.602176634e-19, 'C', 0.0)
(1.67262192369e-27, 'kg', 5.1e-37)
(10973731.56816, 'm^-1', 2.1e-05)
(1.602176634e-19, 'J', 0.0)


Each entry in the dictionary returns three components: the value, the units (as a string), and the experimental uncertainty.

In [4]:
import math
from math import pi

In [5]:
print(pi)

3.141592653589793


### Question 1
Calculate the energy associated with each of the following electromagnetic radiations and arrange them in the increasing order of their energies. (Plank’s constant h = 6.626 x 10-34 J s)
- (a) 200 nm 
- (b) 2500 cm<sup>-1</sup>
- (c) 300 kHz 
- (d) 4000 cm<sup>-1</sup>
- (e) 3 &mu;m

### Answer:

In [6]:
lam_nm = 200            # nm
nm2m = 10**(-9)         # conversion factor from nm -> m
lam_m = lam_nm*nm2m     # m
hnu_j = h*c/lam_m       # J s x m s^1 / m = J

print( '(a) Energy = {:.2e} J'.format( hnu_j ) )

(a) Energy = 9.93e-19 J


In [7]:
lam_cmi = 2500          # cm^-1
lam_cm = 1/lam_cmi      # cm
cm2m = 10**(-2)         # conversion factor from cm -> m
lam_m = lam_cm*cm2m     # m
hnu_j = h*c/lam_m       # J s x m s^1 / m = J

print( '(b) Energy = {:.2e} J'.format( hnu_j ) )

(b) Energy = 4.97e-20 J


In [8]:
lam_hz = 300*10**3      # Hz
lam_cmi = lam_hz/c      # cm^-1
lam_cm = 1/lam_cmi      # cm
cm2m = 10**(-2)         # conversion factor from cm -> m
lam_m = lam_cm*cm2m     # m
hnu_j = h*c/lam_m       # J s x m s^1 / m = J

print( '(c) Energy = {:.2e} J'.format( hnu_j ) )

(c) Energy = 1.99e-26 J


In [9]:
lam_cmi = 4000          # cm^-1
lam_cm = 1/lam_cmi      # cm
cm2m = 10**(-2)         # conversion factor from cm -> m
lam_m = lam_cm*cm2m     # m
hnu_j = h*c/lam_m       # J s x m s^1 / m = J

print( '(d) Energy = {:.2e} J'.format( hnu_j ) )

(d) Energy = 7.95e-20 J


In [10]:
lam_m = 3*10**(-6)      # meter
hnu_j = h*c/lam_m       # J s x m s^1 / m = J

print( '(e) Energy = {:.2e} J'.format( hnu_j ) )

(e) Energy = 6.62e-20 J


Increasing order of energies associated with the electromagnetic radiation:
(c) < (d) < (e) < (b) < (c)

### Question 2
Calculate the speed of electrons emitted from the surface of potassium metal,
when light of wavelength (a) 300 nm (b) 600 nm impinges a clean potassium metal surface?
The work function (&phi;) of potassium is 2.3 eV.

### Answer:
The energy of a photon is calculated using Planck's formula:
E = h &nu; and, &nu; = c &frasl; &lambda; and,

Einstein's photoelectric effect:
$$K.E. = h\nu - \phi$$
$$K.E. = h\frac{c}{\lambda} - \phi$$

To calculate the speed of electrons, what we need is that to calculate the kinetic energy:
$$K.E. = \frac{1}{2}mv^2$$
$$\Rightarrow v^2 = \frac{2\times K.E.}{m} \quad \Rightarrow v = \sqrt{\frac{2\times K.E.}{m}}$$

### Light of wavelength: 300 nm

In [11]:
lam_nm = 300            # nm
nm2m = 10**(-9)         # conversion factor from nm -> m
lam_m = lam_nm*nm2m     # m
hnu_j = h*c/lam_m       # J s x m s^1 / m = J
phi_ev = 2.3            # eV
phi_j = phi_ev*eV       # J
kin_e_j = hnu_j - phi_j
print( 'Kinetic energy = {:.2e} J'.format( kin_e_j ) )

Kinetic energy = 2.94e-19 J


In [12]:
vel = math.sqrt(2*kin_e_j/m_e)                   # sqrt(J/kg) = sqrt(m^2/s^2) = m/s
print( 'Speed = {} m s^-1'.format( vel ) )
print( "Speed = {:.2e} m s^-1".format( vel ) )

Speed = 802941.7101325186 m s^-1
Speed = 8.03e+05 m s^-1


### Light of wavelength: 600 nm

In [13]:
lam_nm = 600            # nm
nm2m = 10**(-9)         # conversion factor from nm -> m
lam_m = lam_nm*nm2m     # m
hnu_j = h*c/lam_m       # J s x m s^1 / m = J
phi_ev = 2.3            # eV
phi_j = phi_ev*eV       # J
kin_e_j = hnu_j - phi_j
print( 'Kinetic energy = {:.2e} J'.format( kin_e_j ) )

Kinetic energy = -3.74e-20 J


Having negative kinetic energy (unreasonable); this simply means that 600 nm photons will not be able to eject electrons from the Potassium metal surface.

### Question 3
Sodium vapour lamps that were widely used as street lights (before the dawn of LED lighting)
emit bright yellow light of wavelength of 590 nm. Calculate the energy separation between
the energy levels in sodium atom responsible for this emission?
Express your answer in joules, in kilojoules per mole, electron volts.

### Answer:
$$E = h\nu$$

In [14]:
lam_nm = 590                     # nm
nm2m = 10**(-9)                  # conversion factor from nm -> m
lam_m = lam_nm*nm2m              # m
hnu_j = h*c/lam_m                # J s x m s^-1 / m => J
hnu_kjm = (hnu_j*10**(-3))*N_A   # kJ/mol
hnu_ev = hnu_j/eV                # eV

print( "Energy separation between energy levels = {:.2e} J".format( hnu_j ) )
print( "Energy separation between energy levels = {:.2f} kJ/mol".format( hnu_kjm ) )
print( "Energy separation between energy levels = {:.2f} eV".format( hnu_ev ) )

Energy separation between energy levels = 3.37e-19 J
Energy separation between energy levels = 202.76 kJ/mol
Energy separation between energy levels = 2.10 eV
